### ISSUE TITLE
Webpack compile error related to 'update-relaunch-buttons-data'

### ISSUE THREAD

**Issue by LukeFinch** [yuanqing/create-figma-plugin#109]
Trying to import the event handlers into my plugin, but running into this error when compiling with webpack.
For context, I'm not using the create-figma-plugin CLI to boostrap my plugin.

Is this something I can fix through my own configs, or is this better addressed in this package?

```ERROR in ./node_modules/@create-figma-plugin/utilities/lib/node/relaunch-button/unset-relaunch-button.js 1:0-104
Module not found: Error: Can't resolve './private/update-relaunch-buttons-data' in '/Volumes/Code/NewsKit/newskit-tokens-ui/applications/figma-plugin/node_modules/@create-figma-plugin/utilities/lib/node/relaunch-button'
Did you mean 'update-relaunch-buttons-data.js'?
BREAKING CHANGE: The request './private/update-relaunch-buttons-data' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
resolve './private/update-relaunch-buttons-data' in '/Volumes/Code/NewsKit/newskit-tokens-ui/applications/figma-plugin/node_modules/@create-figma-plugin/utilities/lib/node/relaunch-button'
  using description file: /Volumes/Code/NewsKit/newskit-tokens-ui/applications/figma-plugin/node_modules/@create-figma-plugin/utilities/package.json (relative path: ./lib/node/relaunch-button)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Volumes/Code/NewsKit/newskit-tokens-ui/applications/figma-plugin/node_modules/@create-figma-plugin/utilities/package.json (relative path: ./lib/node/relaunch-button/private/update-relaunch-buttons-data)
      Field 'browser' doesn't contain a valid alias configuration
      /Volumes/Code/NewsKit/newskit-tokens-ui/applications/figma-plugin/node_modules/@create-figma-plugin/utilities/lib/node/relaunch-button/private/update-relaunch-buttons-data doesn't exist
 @ ./node_modules/@create-figma-plugin/utilities/lib/index.js 30:0-86 30:0-86
 @ ./src/plugin.ts 2:0-52 8:0-2

webpack 5.72.1 compiled with 2 errors in 666 ms
error Command failed with exit code 1.
```

**Comment by yuanqing** [comment]
This is a bug. The `.js` extension is missing in the import path: https://github.com/yuanqing/create-figma-plugin/blob/d9705e9c089560d73bf8314ebe9d2a526459b9f5/packages/utilities/src/node/relaunch-button/set-relaunch-button.ts#L1-L4

The fix for this will be included in the upcoming `2.0.0` release

> Trying to import the event handlers into my plugin

What you could do in the meantime is to specify a direct file path to the `events` module. Something like:

```ts
import { on, once, emit } from '@create-figma-plugin/utilities/lib/events.js'
```

**Comment by LukeFinch** [comment]
Thanks for the quick response, that seems to have worked, although I'm running into a separate issue "figma is not defined" when using the event handlers in the UI context.

(This may be related to our next.js set up, and SSR issues)

Looking forward to the 2.0.0 release and will try again then.
